src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -197,7 +197,7 @@ export interface IStringAxisData {
 export interface INumberAxisData {
     data: {
         uniqueSampleKey: string;
-        value: number | (number[]);
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)